keyboards/keychron/k2/rgb/v2/via_json/k2_ansi.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Keychron K2v2 RGB ANSI",
+    "vendorId": "0x0001",
```

Comment:
```suggestion
    "vendorId": "0x3434",
```

Reply:
should this be changed for the ISO version also?

Reply:
yes, the iso one slipped through. #152 details the keychron specifics for vid/pid. Please do a new PR if you intend to fix that.

Reply:
Changed to appropriate vid. Working fine upon testing.